packages/paste-website/src/pages/components/minimizable-dialog/index.mdx

```diff
@@ -232,12 +232,12 @@ const MinimizableDialogExample: React.FC = () => {
 
 Pass these as part of an object to the useMinimizableDialogState hook.
 
-| Prop         | Type      | Description                                                  | Default                                                                                                                                                                                                                             |
-| ------------ | --------- | ------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
-| `baseId?`    | `string`  | Sets the ID that will serve as a base for all the items IDs. |                                                                                                                                                                                                                                     |
-| `visible?`   | `boolean` | Whether the dialog is visible or not.                        |                                                                                                                                                                                                                                     |
-| `minimized?` | `boolean` | Whether the dialog is minimized or not.                      |                                                                                                                                                                                                                                     |
-| `animated?`  | `number   | boolean`                                                     | If true, animating will be set to true when visible is updated. It'll wait for stopAnimation to be called or a CSS transition ends. If animated is set to a number, stopAnimation will be called after that number of milliseconds. |     |
```

Comment:
issue: the table below this one suffers from the same error too.

Reply:
Good catch, fixed